### ISSUE TITLE
Add Documentation

### ISSUE THREAD

**Issue by vjspranav** [vjspranav/timetable#3]
Help mcreate documentation on how to run
* Simple react app npm start

**Comment by Devenkapoor7303** [comment]
Hey can I work on this issue can you assign it to me.

**Comment by debodyuti1** [comment]
Hey I can work on this!

**Comment by debodyuti1** [comment]
Do you want me to add the documentation to a README.md file?

**Comment by vjspranav** [comment]
> Do you want me to add the documentation to a README.md file?

Yes. with steps on how too deploy and run. It is quite generic, like all react projects.

**Comment by Devenkapoor7303** [comment]
I thought that issues were assigned on the first come first basis?

**Comment by vjspranav** [comment]
> I thought that issues were assigned on the first come first basis?

There is no professional way. I am assigning to whoever is willing to. I'll merge the first request that meet specifications, that comes my way.

**Comment by Devenkapoor7303** [comment]
Okay

**Comment by debodyuti1** [comment]
@vjspranav did you check my pull request?
